dev/WebView2/WebView2.cpp

```diff
@@ -1763,7 +1792,8 @@ void WebView2::CheckAndUpdateWebViewPosition()
         return;
     }
 
-    // Check if the position of the WebView within the window has changed
+    // Check if the position of the WebView2 within the window has changed
+    bool changed = false;
```

Comment:
Great to see this change!

Reply:
Also to clarify - this change is a cleanup/performance improvement, not strictly necessary to address the bug with WV2 starting invisible, right?

Reply:
Yes, it's a separate thing I noticed while working on this.